scripts/twister

```diff
@@ -584,6 +584,12 @@ structure in the main Zephyr tree: boards/<arch>/<board_name>/""")
              "This directory will be cleaned unless '--no-clean' is set. "
              "The '--clobber-output' option controls what cleaning does.")
 
+    parser.add_argument(
```

Comment:
please put this argument in alpha-beta order, this is starting from `s`, and shall not put before `o`

Reply:
Oh, yes, you're right, I replaced it in correct place :)